### ISSUE TITLE
hope support lcm

### ISSUE THREAD

**Issue by mliand** [ArtVentureX/comfyui-animatediff#63]
hope support lsm https://huggingface.co/latent-consistency

**Comment by artventuredev** [comment]
Yes LCM LoRA is supported out of the box. Here's a simple workflow with LCM included: [simple-lcm.json](https://github.com/ArtVentureX/comfyui-animatediff/files/13328017/simple-lcm.json)
